marimo/_cli/convert/markdown.py

```diff
@@ -42,35 +43,60 @@ def _is_code_tag(text: str) -> bool:
     return bool(re.search(r"\{.*python.*\}", head))
 
 
-def formatted_code_block(code: str) -> str:
+def formatted_code_block(
```

Comment:
sometime this week - i may write some small unit tests for these files (as my way to learn this part of the code more)

Reply:
🛳️